src/libraries/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -163,7 +169,14 @@ internal static Color ArgbToKnownColor(uint argb)
             {
                 if (s_colorTable[index] == argb)
                 {
-                    return Color.FromKnownColor((KnownColor)(index + (int)KnownColor.Transparent));
+                    var knownColor = KnownColor.Transparent + index;
+
+                    if (knownColor > KnownColor.YellowGreen)
```

Comment:
Maybe add a comment on why this is needed?

Reply:
i added a comment to it. should i also add a comment to the other ones? like on these codes:
```suggestion
                   return color < KnownColor.ButtonFace
                 ? s_colorTable[(int)color - (int)KnownColor.Transparent]
                 : s_colorTable[(int)color - (int)KnownColor.RebeccaPurple + ((int)KnownColor.YellowGreen - (int)KnownColor.WindowText)];
```

Reply:
I think it is good with the one you added, thanks.